### PR TITLE
Adds Sofle choc

### PR DESCRIPTION
**Pull request by JellyTitan**
The Sofle Choc has a separate directory from the other Sofle keyboards in QMK. I think it makes sense to maintain the separation here as well. 
image

## Checklist

- [x] The VIA support for this keyboard is **MERGED** in QMK master already **(MANDATORY)**
- [x] The VIA definition follows the guide here: https://caniusevia.com/docs/layouts
- [x] I have a V3 JSON version for this keyboard definition.**(MANDATORY)**
- [x] I have tested this keyboard definition using VIA's "Design" tab.
- [x] I have tested this keyboard definition with firmware on a device.
- [x] I have assigned alpha keys and modifier keys with the correct colors.
- [x] The Vendor ID is not `0xFEED`

Hurray! It works!
Verified Sofle Pico VIA support

### CODE REVIEW

File: v3/sofle_pico/sofle_pico.json [hunk 1]
```diff
@@ -0,0 +1,39 @@
+{
+  "name": "Sofle Pico",
```
Comment (Xelus22) [inline v3/sofle_pico/sofle_pico.json:2]: Sofle Pico doesnt exist in QMK master?

**Review by Xelus22 — CHANGES_REQUESTED** [review]

File: v3/sofle_pico/sofle_pico.json (hunk 1 above)
Comment (JellyTitan) [reply v3/sofle_pico/sofle_pico.json]: How embarrassing!  That was another board in development. I removed it. Thank you for catching that!

File: v3/sofle_choc/sofle_choc.json [hunk 2]
```diff
@@ -0,0 +1,39 @@
+{
+  "name": "Sofle Choc",
+  "vendorId": "0xFC32",
+  "productId": "0x0287",
```
Comment (Xelus22) [inline v3/sofle_choc/sofle_choc.json:4]: This conflicts with `sofle/rev1`

**Review by Xelus22 — CHANGES_REQUESTED** [review]

File: v3/sofle_choc/sofle_choc.json (hunk 2 above)
Comment (Xelus22) [reply v3/sofle_choc/sofle_choc.json:4]: https://github.com/qmk/qmk_firmware/blob/master/keyboards/sofle/rev1/info.json#L7-L8

Comment (JellyTitan) [reply v3/sofle_choc/sofle_choc.json:4]: Good catch! I opened an issue with Sofle repo to see if [we can fix that duplicate ID](https://github.com/josefadamcik/SofleKeyboard/issues/194#issuecomment-1942966178).

Comment (JellyTitan) [reply v3/sofle_choc/sofle_choc.json:4]: The sofle_choc creator, Brian Low, [approved a PID/VID change](https://github.com/josefadamcik/SofleKeyboard/issues/194#issuecomment-1943001616).
I opened a [PR with QMK to make the sofle_choc PID/VID unique](https://github.com/qmk/qmk_firmware/pull/23080#issue-2134483702).

Comment (JellyTitan) [reply v3/sofle_choc/sofle_choc.json:4]: I goofed and closed that initial QMK PR. A new one has been opened with the same vid/pid fix:
https://github.com/qmk/qmk_firmware/pull/24030

Comment (JellyTitan) [reply v3/sofle_choc/sofle_choc.json]: PR accepted - hurray!

**Comment by JellyTitan** [comment]
The PR to make the Sofle Choc PId/Vid Unique was accepted and merged into QMK master. 
I updated the Sofle Choc VIA PR to match the update VID/PID.
Ready for review!

**Review by Xelus22 — APPROVED** [review]

**Comment by Xelus22** [comment]
https://github.com/qmk/qmk_firmware/pull/24030

**Pull request merged**